studio/components/interfaces/Organization/OAuthApps/OAuthApps.tsx

```diff
@@ -0,0 +1,193 @@
+import { useParams } from 'common'
+import Table from 'components/to-be-cleaned/Table'
+import { OAuthApp, useOAuthAppsQuery } from 'data/oauth/oauth-apps-query'
+import { useState } from 'react'
+import { Alert, Button, IconClipboard, IconEdit, IconTrash, IconX, Input } from 'ui'
+import PublishAppModal from './PublishAppModal'
+import DeleteAppModal from './DeleteAppModal'
+import ShimmeringLoader from 'components/ui/ShimmeringLoader'
+import AlertError from 'components/ui/AlertError'
+import { copyToClipboard } from 'lib/helpers'
+import OAuthAppRow from './OAuthAppRow'
+import { OAuthAppCreateResponse } from 'data/oauth/oauth-app-create-mutation'
+
+// [Joshen] Just FYI need to relook at the copy writing at the end
+
+const OAuthApps = () => {
+  const { slug } = useParams()
+  const [createdApp, setCreatedApp] = useState<OAuthAppCreateResponse>()
+  const [showPublishModal, setShowPublishModal] = useState(false)
+  const [selectedAppToUpdate, setSelectedAppToUpdate] = useState<OAuthApp>()
+  const [selectedAppToDelete, setSelectedAppToDelete] = useState<OAuthApp>()
+
+  const {
+    data: publishedApps,
+    isLoading,
+    isSuccess,
+    isError,
+  } = useOAuthAppsQuery({
+    slug,
+    type: 'published',
+  })
+
+  const sortedPublishedApps = publishedApps?.sort((a, b) => {
+    return Number(new Date(a.created_at)) - Number(new Date(b.created_at))
+  })
+
+  return (
+    <>
+      <div className="container my-4 max-w-4xl space-y-8">
+        <div>
+          <div className="flex items-center justify-between">
+            <div>
+              <p>Published Apps</p>
+              <p className="text-scale-1000 text-sm">
+                Build on top of the Supabase platform by having access to your organization's
+                settings and projects
```

Comment:
Hmm it would have access to anyone who gives it permission so it's not just limited to your own org / projects. Maybe just a short description like this will do? We'll have a more descriptive guide on how to use / create an oauth app too.
```suggestion
                Build integrations that extend Supabase's functionality.
```

Reply:
ahh yup agreed - this sounds good